e2e/preview/test_prompt.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+import pytest
+
+from haystack.preview.components.prompt import Prompt
+
+
+# FIXME hangs locally!
```

Comment:
What's the problem with the huggingface remote inference here?

Reply:
It needs an HF token.